SPECS/perl-PadWalker/perl-PadWalker.spec

```diff
@@ -1,6 +1,6 @@
 Name:           perl-PadWalker
 Version:        2.3
```

Comment:
upgrade to 2.5 is available

Reply:
Updated PadWalker and Test-Without-Modules.
Sub-Identify is already at the latest version.
Buddy build: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=631631&view=results